src/docs/index.md

```diff
@@ -60,25 +60,18 @@ Coming from another platform? Check out:
 
 ### Videos
 
-We also have some helpful videos on our
-[Flutter YouTube channel][]! In particular, check
-out the Flutter in Focus series,
-and learn about other series on our [videos][] page.
+We also have some helpful videos on our [Flutter YouTube channel][]! In particular, check out the [Flutter in Focus series][], and learn about other series on our [videos][] page.
 
-First up, why use Flutter? What makes it different than other
-app frameworks?
+First up, why use Flutter? What makes it different than otherapp frameworks?
```

Comment:
otherapp => other app

Reply:
Okay, I will make another commit, Thanks for the review!

Reply:
hey @sfshaza2, I have made the changes as you told me to do, kindly take a look into it. And Thank you very much to give such great insight into the issue.